obspy/imaging/maps.py

```diff
@@ -468,7 +468,7 @@ def linspace2(val1, val2, n):
     # scatter plot is removing valid x/y points with invalid color value,
     # so we plot those points separately.
     try:
-        nan_points = np.isnan(np.array(color, dtype=np.float))
+        nan_points = np.isnan(np.array(color, dtype=float))
```

Comment:
Would it be better to specify a `np.float64` rather than native float type?

Reply:
Maybe? But these were aliases, so this just fixes the warning without making a choice on the matter.

Reply:
@QuLogic what you mean is:
currently; on a 32bits system, np.float was just an alias for native float32 ; and on a 64bits system, np.float was float64 ? right ?

Reply:
No, what I meant is that `np.float` is an alias for `float`, and @d-chambers is asking about what you mentioned. 

From testing it out though, using `float` is always 64-bit, even on a 32-bit build (tried on Linux, don't know about Windows).

Reply:
It _does_ have an effect on `int`, which is 32-bit on 32-bit and 64-bit on 64-bit, but that's the same as `np.int`, because they're the same thing.

Reply:
Yes, I have run into `int` issues on windows before and had to specify np.int64. Good point though, you aren't changing any behavior though, so its probably fine. I will merge.